OSDOCS-21049: Update WIF permission docs for Milestone-2 least-privilege changes (#117548)
openshift/openshift-docs: modules/create-wif-configuration.adoc

= Creating a Workload Identity Federation configuration

[role="_abstract"]
You can create a Workload Identity Federation (WIF) configuration using the `auto` mode or the `manual` mode in the `ocm` CLI.

In `auto` mode, the service accounts for {product-title} components and other IAM resources are created automatically.

> Alternatively, you can use the `manual` mode. In `manual` mode, a `script.sh` file provides commands that you use to manually create the service accounts for {product-title} components and other IAM resources.

Reviewer: I believe IBM SG wants us to avoid "Alternatively" 

What do you think of this? 

```suggestion
You can also use the `manual` mode where a `script.sh` file has commands that you can use to manually create the service accounts for {product-title} components and other IAM resources.
```
